File: testers/combined/java/generated-and-checked-in/api/combined/api/api/ProductsApi.java
package combined.api.api;

import combined.api.model.Product;
import io.smallrye.mutiny.Uni;
import java.util.List;
import java.util.Optional;

public interface ProductsApi {
  /** List all products from both databases */
  Uni<List<Product>> listProducts(
    /** Filter by data source */
    Optional<String> source,
    /** Filter by active status */
    Optional<Boolean> isActive
  );
}

File: testers/combined/kotlin/generated-and-checked-in/api/combined/api/api/ProductsApi.kt
package combined.api.api

import combined.api.model.Product
import io.smallrye.mutiny.Uni
import kotlin.collections.List

interface ProductsApi {
  /** List all products from both databases */
  abstract fun listProducts(
    /** Filter by data source */
    source: kotlin.String?,
    /** Filter by active status */
    isActive: kotlin.Boolean?
  ): Uni<List<Product>>
}